ftw/referencewidget/behaviors.py

```diff
@@ -0,0 +1,78 @@
+from ftw.referencewidget.sources import ReferenceObjSourceBinder
+from ftw.referencewidget.widget import ReferenceWidgetFactory
+from plone.app.dexterity import MessageFactory as _
+from plone.autoform import directives
+from plone.autoform.interfaces import IFormFieldProvider
+from plone.supermodel.interfaces import FIELDSETS_KEY
+from plone.supermodel.model import Fieldset
+from z3c.relationfield.schema import RelationChoice, RelationList
+from zope.interface import alsoProvides
+from zope.interface import Interface
+
+
+class IRelatedItems(Interface):
+    """Behavior, which provides the same functionality as the
+    plone.app.relationfield IRelatedItems behavior, BUT it uses a different
+    widget.
+
+    """
+    directives.widget(relatedItems=ReferenceWidgetFactory)
+    relatedItems = RelationList(
+        title=_(u'label_related_items', default=u'Related Items'),
+        default=[],
+        value_type=RelationChoice(title=u"Related",
+                                  source=ReferenceObjSourceBinder()),
+        required=False,
+    )
+
+fieldset = Fieldset('categorization',
+                    label=_(u'Categorization'),
+                    fields=['relatedItems'])
+IRelatedItems.setTaggedValue(FIELDSETS_KEY, [fieldset])
```

Comment:
It's a exact copy of plone.app.relationfield.IRelatedItems behavior. 
Gonna change that. 👍